numba/cpython/randomimpl.py

```diff
@@ -1299,7 +1299,12 @@ def random_arr(context, builder, sig, args, typing_key=typing_key):
         arr = arrayobj._empty_nd_impl(context, builder, arrty, shapes)
 
         # ... and populate it in natural order
-        scalar_impl = context.get_function(typing_key, scalar_sig)
+        *mod, fname = typing_key.split('.')
+        np_func = getattr(np.random, fname)
```

Comment:
`mod` is unused and `np.random` is hardcode in `getattr`. Maybe there should be a assert to make sure `mod` is `np.random`

Reply:
Good point, done in 332c999